File: 240809/코드네임/code-name.cpp
#include <iostream>
#include <string>

using namespace std;

class Person {
public:
    string name;
    int score;

    // 생성자를 public으로 선언
    Person(string name = "", int score = 0) {
        this->name = name;
        this->score = score;
    }
};

int main() {
    Person persons[5];

    // 5명의 요원 정보를 입력받음
    for(int i = 0; i < 5; i++) {
        string name;
        int score;

        cin >> name >> score;

        persons[i] = Person(name, score);
    }

    // 점수가 제일 낮은 요원을 찾기 위한 초기화
    Person minPerson = persons[0];

    for(int i = 1; i < 5; i++) {
        if(persons[i].score < minPerson.score) {
            minPerson = persons[i];
        }
    }

    // 점수가 제일 낮은 요원의 정보 출력
    cout << minPerson.name << ' ' << minPerson.score << '\n';

    return 0;
}

File: 241019/코드네임/code-name.py
class person:
    def __init__(self, name="", score=0) :
        self.name = name
        self.score = score

people = []

for i in range(5) :
    n, s = tuple(input().split())
    s = int(s)
    people.append(person(n, s))

mn = 100
ans = 0

for i in range(5) :
    if people[i].score < mn :
        mn = people[i].score
        ans = i

print(people[ans].name, people[ans].score)